src/domain/event/eventInfo/__tests__/EventInfo.test.tsx

```diff
@@ -364,3 +391,225 @@ describe('OrganizationInfo', () => {
     }
   );
 });
+
+describe('superEvent', () => {
+  it('should render super event title and link when super event is given', async () => {
+    const superEvent = fakeEvent({
+      superEvent: { internalId: superEventInternalId },
+    });
+    const superEventResponse = {
+      data: superEvent,
+      status: 'resolved',
+    } as SuperEventResponse;
+    const { history } = render(
+      <EventInfo
+        event={event}
+        superEvent={superEventResponse}
+        eventType="event"
+      />,
+      {
+        mocks,
+      }
+    );
+    await actWait();
+    expect(
+      screen.queryByRole('heading', {
+        name: translations.event.superEvent.title,
+      })
+    ).toBeInTheDocument();
+
+    userEvent.click(
+      within(screen.getByTestId(superEventTestId)).queryByText(
+        superEvent.name.fi
+      )
+    );
+    expect(history.location.pathname).toBe(`/fi/events/${superEvent.id}`);
+  });
+
+  it('should should not render super event title when super event is not given', async () => {
+    render(<EventInfo event={event} eventType="event" />, {
+      mocks,
+    });
+    await actWait();
```

Comment:
Nothing asyncronous happening in this case? Or is it added to be futureproof somehow? :)